internal/cmd/egctl/testdata/translate/out/jwt-single-route-single-match-to-xds.all.json

```diff
@@ -393,7 +393,7 @@
                       ],
                       "loadBalancingWeight": 1,
                       "locality": {
-                        "region": "raw_githubusercontent_com_443/backend/0"
+                        "region": "raw_githubusercontent_com_443/backend/-1"
```

Comment:
why there's `-1`, is it should be `0`?

Reply:
I've used -1 as an index for all settings which are based off a singluar field like backendRef ( and not backendRefs)